doc/releases/release-notes-3.6.rst

```diff
@@ -285,6 +310,10 @@ Drivers and Sensors
 
 * PCIE
 
+  * Fixed MMIO size calcultion by disabling IO/memory decoding beforehand.
```

Comment:
your keyboard doesn't like that word! :)

```suggestion
  * Fixed MMIO size calculation by disabling IO/memory decoding beforehand.
```

Reply:
Hahaha... I got a new keyboard a few days ago and am still getting used to it. :)